ts/masters/indicators/indicators.src.ts

```diff
@@ -11,4 +11,5 @@
  * License: www.highcharts.com/license
  */
 'use strict';
+import '../../Stock/Indicators/EMA/EMAIndicator.js';
```

Comment:
Q: Does the order not matter here?

Reply:
Outside of karma it probably does. Fixed.